### PR TITLE
Add 1.1.1.1 (CloudFlare) to DNS server list

### PR DESCRIPTION
**Pull request by pfgithub**

### CODE REVIEW

File: auto_install/install.sh [hunk 1]
```diff
@@ -734,6 +735,13 @@ setClientDNS() {
             $SUDO sed -i '0,/\(dhcp-option DNS \)/ s/\(dhcp-option DNS \).*/\1'${OVPNDNS1}'\"/' /etc/openvpn/server.conf
             $SUDO sed -i '0,/\(dhcp-option DNS \)/! s/\(dhcp-option DNS \).*/\1'${OVPNDNS2}'\"/' /etc/openvpn/server.conf
             ;;
+        CloudFlare)
+            echo "::: Using CloudFlare servers."
+            OVPNDNS1="1.1.1.1"
+            OVPNDNS2="1.0.0.1"
+            $SUDO sed -i '0,/\(dhcp-option DNS \)/ s/\(dhcp-option DNS \).*/\1'${OVPNDNS1}'\"/' /etc/openvpn/server.conf
+            $SUDO sed -i '0,/\(dhcp-option DNS \)/! s/\(dhcp-option DNS \).*/\1'${OVPNDNS2}'\"/' /etc/openvpn/server.conf
```
Comment (redfast00) [inline auto_install/install.sh]: While you're at it, can you remove these duplicate lines of code?

**Comment by redfast00** [comment]
@pfgithub this has been solved in another PR, can you please fix the PR?

**Comment by pfgithub** [comment]
@redfast00 Updated

**Comment by redfast00** [comment]
Thanks!

**Pull request merged**